### PR TITLE
Login register page

### PR DESCRIPTION
**Pull request by VovaProtsiv**

### CODE REVIEW

File: src/app/auth/auth.service.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+const httpOptions = {
+  headers: new HttpHeaders({ 'Content-Type': 'application/json' })
+};
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthService {
+
+  private loginUrl = 'http://localhost:8080/api/v1/auth/login';
```
Comment (Forfend) [inline src/app/auth/auth.service.ts:18]: create a file with constants

File: src/app/auth/auth.service.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import {AuthLoginInfo} from "./auth-login-info";
```
Comment (Forfend) [inline src/app/auth/auth.service.ts:4]: use ```''```

File: src/app/auth/token-storage.service.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
+import { Injectable } from '@angular/core';
+import  jwt_decode from 'jwt-decode';
+const TOKEN_KEY = 'AuthToken';
+const USERNAME_KEY = 'AuthUsername';
```
Comment (Forfend) [inline src/app/auth/token-storage.service.ts:4]: these constants should be in a single util file, that contains all the constants for project

**Review by Forfend — CHANGES_REQUESTED** [review]

File: src/app/auth/auth.service.ts (hunk 2 above)
Comment (VovaProtsiv) [reply src/app/auth/auth.service.ts]: done

File: src/app/auth/token-storage.service.ts (hunk 3 above)
Comment (VovaProtsiv) [reply src/app/auth/token-storage.service.ts]: defined clobal constants in common/AppConfig.ts

File: src/app/auth/auth.service.ts (hunk 1 above)
Comment (VovaProtsiv) [reply src/app/auth/auth.service.ts]: defined clobal constants in common/AppConfig.ts

**Pull request merged**